src/pages/Home/Home.js

```diff
@@ -10,6 +10,10 @@ export default function Home() {
   const history = useHistory();
   const navigateToSearchPage = (q) => {
     console.log(q);
```

Comment:
Do you want to remove or comment out all console logging?

Reply:
Good point; removed this and the logging from a few extra places.